AWSCore/UICKeyChainStore/AWSUICKeyChainStore.m

```diff
@@ -935,6 +935,26 @@ + (NSArray *)prettify:(CFTypeRef)itemClass items:(NSArray *)items
 
 #pragma mark -
 
+- (void)migrateToCurrentAccessibility {
+    NSArray *items = [self allItems];
+    for (NSDictionary *item in items) {
+        CFComparisonResult result = CFStringCompare((CFStringRef)item[@"accessibility"],
+                                                    [self accessibilityObject], 0);
+        if (result == 0) {
+            continue;
+        }
+        NSString *key = item[@"key"];
+        NSObject *value = item[@"value"];
+        if ([value isKindOfClass:[NSString class]]) {
+            [self setString: (NSString *)value forKey:key];
+        } else {
+            [self setData: (NSData *)value forKey:key];
```

Comment:
Same question as above re: the type of incoming `value`.

Reply:
Updated